lib/src/rover/orchestrator.dart

```diff
@@ -39,11 +39,13 @@ class RoverOrchestrator extends OrchestratorInterface with ValueReporter {
     collection.logger.info("Got GPS Task: Go to ${destination.prettyPrint()}");
     collection.logger.debug("Currently at ${collection.gps.coordinates.prettyPrint()}");
     collection.drive.setLedStrip(ProtoColor.RED);
+    collection.detector.findObstacles();
+    // await collection.drive.faceNorth();
```

Comment:
You'll have to either add this back in, or change the A* to take into account the initial orientation of the rover.